File: RoadBuilder/UI/src/domain/NetSectionItem.ts
import { LaneSectionType } from "./LaneSectionType";

export interface NetSectionItem {
  PrefabName: string;
  DisplayName: string;
  Thumbnail: string;
  IsGroup: boolean;
  IsEdge: boolean;
  IsRestricted: boolean;
  IsCustom: boolean;
  Width?: number;
  WidthText?: string;
  SectionType: LaneSectionType;
}

File: RoadBuilder/Domain/UI/NetSectionItem.cs
using RoadBuilder.Domain.Enums;

namespace RoadBuilder.Domain.UI
{
	public class NetSectionItem
	{
		public string? PrefabName;
		public string? DisplayName;
		public string? Thumbnail;
		public bool IsGroup;
		public bool IsEdge;
		public bool IsRestricted;
		public bool IsCustom;
		public float Width;
		public string? WidthText;
		public LaneSectionType SectionType;
	}
}
